### ISSUE TITLE
Add search to User page

### ISSUE THREAD

**Issue by mattlindsey** [AgileVentures/WebsiteOne#1749]
Add search functionality by user_name to Members page

**Comment by tansaku** [comment]
```
@pauldev_eu Simple,
@praveendhawan Medium,
@mlindsey Medium,
@tansaku Medium
4 vote(s) so far [@pauldev_eu,@praveendhawan,@mlindsey,@tansaku]


Sam Joseph
[12:11 PM] 
@pauldev_eu_ what's your reasoning behind simple versus medium?

Paul Andrei [12:20 PM] 
@tansaku after reviewing the code that handles the current search feature I've came to the conclusion that only minimal changes would be needed on implementing this.

First of, set_filter_params would need to be modified to include either the first name, last name or both. And then another change on the view for the text search form.

I can't really see any other change that might be needed in order to make this work, but I'm open to better suggestions!


Sam Joseph
[12:20 PM] 
@mlindsey and I finished the scrum - we checked that the existing search (via project etc.) appears to use ajax, and that was probably our motivation for making it medium vs simple ...  although we agreed it could turn out to be easy, and agreed we were being conservative (edited)


[12:20] 
your analysis looks correct


[12:21] 
I'd be happy to adjust to simple - how about you @praveendhawan and @mlindsey ?


Matt Lindsey
[12:22 PM] 
@tansaku simple is good!


Praveen Dhawan [12:33 PM] 
What if we give live search in the select box, with every typed keyword it will fetch the result via ajax. That would make it medium


[12:34] 
But as you say it can be simple


[12:34] 
So I wasn't sure of the method by which we are going to solve this. That's why I kept it at medium


Sam Joseph
[12:41 PM] 
yeah, it's an interesting question as to whether we should do "live search" - that's what was previously implemented for search by name, when all the users where shown in one page ...


[12:41] 
maybe it's best to implement non-live search (simple) first, and then have "live search" as a subsequent improvement?


Sam Joseph
[12:42 PM] 
would love infinite scroll, but knows that some folks like @b264 actually prefer pagination ...


Paul Andrei [12:44 PM] 
@praveendhawan @tansaku I also agree that live search could be a good option.


[12:49] 
we should wrap up this vote - @here - shall we agree that this one is simple, and we can open a separate ticket for live search once this is done?



Paul Andrei [12:50 PM] 
@tansaku I'd say so. 
Live search sounds like a good option but it might be even better to do it bit by bit and see how this feature evolves? So I say that we leave it at simple and open a separate ticket for live search as an improvement on top of what will be built now.
```

**Comment by mattlindsey** [comment]
+1
